specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/examples/P2SVpnGatewayGet.json

```diff
@@ -2,7 +2,7 @@
   "parameters": {
     "gatewayName": "p2sVpnGateway1",
     "resourceGroupName": "rg1",
-    "api-version": "2019-09-01",
+    "api-version": "2019-08-01",
```

Comment:
Wrong API version (in this and other examples)

Reply:
Corrected.